app/controllers/api/v1/emotions_controller.rb

```diff
@@ -70,6 +71,23 @@ def emotion_params
     params.require(:emotion).permit(:word, :category)
   end
 
+  def fun_question
+    custom_question.presence || FunQuestion.question_public.not_used.order('RANDOM()').first
```

Comment:
I think `FunQuestion.question_public.not_used.sample` would be a more standard way of doing this.